service/controller/resource/tccp/template/template_main_vpc.go

```diff
@@ -12,6 +12,11 @@ const TemplateMainVPC = `
       Tags:
       - Key: Name
         Value: {{ $v.ClusterID }}
+  VPCCIDRBlockAWSCNI:
+    Type: AWS::EC2::VPCCidrBlock
+    Properties:
+      CidrBlock: {{ $v.CIDRBlockAWSCNI }}
```

Comment:
from above comment this could be something like {{ $v.CNISubnet }}/{{ $v.CNICIDR }}